src/Azure.Functions.Cli/Common/AuthSettingsFile.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Azure.Functions.Cli.Common;
+using Colors.Net;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+namespace Azure.Functions.Cli.Common
+{
+    class AuthSettingsFile
+    {
+        public bool IsEncrypted { get; set; }
+        public Dictionary<string, string> Values { get; set; } = new Dictionary<string, string>();
```

Comment:
Make this a case-insensitive dictionary: new `Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)`. That way you don't have to remember to use case-sensitive lookups every time you reference this.